.github/workflows/format.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.58.1
+          toolchain: nightly-2022-04-07
```

Comment:
Is this change needed? I hope we could achieve:
1) build components which will run inside tdx with nightly
2) build components which will run on host side with stable

Reply:
ok, changed it back